rust_src/build.rs

```diff
@@ -77,10 +77,7 @@ impl ModuleInfo {
             let tmp = path_as_str(mod_path.file_name()).to_string();
             let path = mod_path.join("mod.rs");
             if path.is_file() {
-                return Some(ModuleInfo {
-                    path: path,
-                    name: tmp,
-                });
+                return Some(ModuleInfo { path, name: tmp });
```

Comment:
Might as well rename `tmp` to `name` as well so it has a semantically meaningful and can be shortened too, if that's not outside the scope of this PR

Reply:
That sounds reasonable, let me look at it.